Sources/PointFree/AtomFeed.swift

```diff
@@ -153,7 +167,7 @@ that day!
     description: description(episode: episode),
     dublinCore: .init(creators: ["Brandon Williams", "Stephen Celis"]),
     enclosure: enclosure(episode: episode),
-    guid: url(to: .episode(.left(episode.slug))),
+    guid: String(describing: episode.freeSince ?? episode.publishedAt),
```

Comment:
using the episodes free date / published date as the GUID allows us to create a new entry for episodes that were subscriber only and then switched to free.

Reply:
Is `String(describing:)` subject to weird local/locale formatting? Should we use a `DateFormatter` to be safe for testing?

Reply:
oh im not sure. I could also just use the underlying time interval. how's that sound?

Reply:
Oh yeah that sounds perf!

Reply:
coo done 5fd134fe